Makefile

```diff
@@ -15,8 +15,8 @@ CFLAGS := -g -O2 -std=c99 -Wall -Werror -Wextra -Wpedantic -Wundef -Wconversion
 CC := cc
 WASI_SDK_PATH := /opt/wasi-sdk
 
-HEADERS := $(shell find include -name '*.h')
-SOURCES := $(shell find src -name '*.c')
```

Comment:
`Makefile` doesn’t support recursive `wildcard`, so shelling out was accustomed.
The alternative is to write a custom function.

I’m fine with abandoning this change if it’s not worth to not shell-out.

For reference, `mkmf`-generated `Makefile`s does __not__ expand recursively; that is, `mkmf` does not support source code subfolders.